llvm/tools/llvm-exegesis/lib/MCInstrDescView.cpp

```diff
@@ -166,6 +166,9 @@ Instruction::create(const MCInstrInfo &InstrInfo,
   BitVector ImplUseRegs = RATC.emptyRegisters();
   BitVector AllDefRegs = RATC.emptyRegisters();
   BitVector AllUseRegs = RATC.emptyRegisters();
+  BitVector MemoryRegs = RATC.emptyRegisters();
```

Comment:
It seems that `MemoryRegs` is not used. Do we need it?

Reply:
Not currently used, it was added in addition to NotMemoryRegs since they are all aliased registers together.

Reply:
I think we should remove it until it is used.

Reply:
Addressed